test/KoreBuild.FunctionalTests/SimpleRepoTests.cs

```diff
@@ -51,8 +51,7 @@ public SimpleRepoTests(ITestOutputHelper output, RepoTestFixture fixture)
 
             using (var reader = new PackageArchiveReader(libPackage))
             {
-                Assert.Contains("lib/netstandard2.0/Simple.Lib.pdb", reader.GetFiles());
-                Assert.Contains("lib/net461/Simple.Lib.pdb", reader.GetFiles());
+                Assert.Empty(reader.GetFiles().Where(p => Path.GetExtension(p).Equals(".pdb", StringComparison.OrdinalIgnoreCase)));
```

Comment:
My usual comment: always call static `string.Equals` because it properly guards against `null`. Not sure what GetExtension returns (nor do I trust it!).